src/plat/bcm2711/overlay-rpi4.dts

```diff
@@ -37,18 +37,13 @@
 	 * 0x01 0x00000000 0x80000000  2048MB
 	 * 0x01 0x80000000 0x80000000  2048MB
 	 *
+	 * There are multiple models of the RPi4B with different sizes of RAM.
+	 * The above outlines the memory ranges for the 8GB model, for models
+	 * with less than 4GB of RAM, the last two ranges are omitted. For the
+	 * 2GB model, the range starting at 0x40000000 is smaller in size.
```

Comment:
I reckon this should explicitly state that in "low peripheral" mode there is a reserved region [0xfc000000, 0x100000000)` that affects models with 4GiB or more of RAM (I assume there's just a hole of inaccessible memory in the RAM device, but is this even confirmed anywhere?).

Reply:
YEs, the linked threads do explain that the peripherals are mapped over the top of the RAM so the 64MiB is "lost"

Reply:
Yes I believe it's explained in the TRM, should I make it clearer in the comment or leave it as it is?

Reply:
Can you update the comment to state "In 'low peripheral' mode there is a reserved region [0xfc000000, 0x100000000) for device peripheral mappings"

Reply:
Done

Reply:
Thanks